src/main/resources/rs117/hd/scene/model_overrides.json

```diff
@@ -1342,15 +1341,18 @@
   {
     "description": "TREE_STUMP",
     "baseMaterial": "BARK",
+    "castShadows": false,
```

Comment:
It seems a little bit weird to me to disable shadow casting for [all of these tree stumps](https://chisel.weirdgloop.org/moid/object_id.html#1341,1342,1353,4328,6212,9661,9712,14516,14517,50089). Do you have any particular reason behind this?

Reply:
Whoops I didn't mean to leave that in, I noticed some of the stumps were casting a shadow that was visible underneath the model because jagex placed them on a slope tile so I was messing around with that to see if it was an improvement. 

Feel free to snip that out.